app/models/reviewable_flagged_post.rb

```diff
@@ -47,6 +47,10 @@ def build_actions(actions, guardian, args)
     agree =
```

Comment:
The current bundle names like `agree` are pretty confusing, I was searching for what that meant before I realised it was the bundle being passed to `delete_user_actions`. Maybe the result of anything calling `add_bundle` should be called `x_bundle` e.g. `agree_bundle`?